Phone book/Phone book/phoneBook.c

```diff
@@ -0,0 +1,171 @@
+#include <stdio.h>
+#include <string.h>
+
+#include "phoneBook.h"
+
+typedef struct Note
+{
+	char name[MAX_NAME_LENGTH];
+	char phoneNumber[MAX_PHONE_NUMBER_LENGTH];
+} Note;
+
+struct PhoneBook
+{
+	Note notes[MAX_CONTACTS_NUMBER];
+	size_t numberOfContacts;
+};
+
+PhoneBook* createPhoneBook(void)
+{
+	return (PhoneBook*)calloc(1, sizeof(PhoneBook));
+}
+
+ErrorCode initData(PhoneBook* const phoneBook, const char* const openningFileName)
```

Comment:
```suggestion
ErrorCode initData(PhoneBook* const phoneBook, const char* const fileNameToOpen)
```
Или как-то так, иначе как-то не по-английски